### ISSUE TITLE
how to access the HTMLElement of Button?

### ISSUE THREAD

**Issue by koo5** [bestguy/sveltestrap#275]
i'm trying: 
```<Button use:xxx on:click={more}>more...</Button>```
and `xxx()` is called with what seems to be an empty object.
i'm trying:
```<Button bind:this={my_button} on:click={more}>more...</Button>```
and whatever `my_button` is set to, calling blur() on it (a HTMLElement method) has no effect.
To the contrary, with:
```<button bind:this={my_button} on:click={more}>more...</button>```
i can call my_button.blur() and it correctly removes focus from the element.

**Comment by bestguy** [comment]
Svelte does not support `use:xxx` on custom elements, only built-in DOM elements.
With that said, your need is to `bind:this` to the button element and manually call blur, correct?

**Comment by koo5** [comment]
right!

**Comment by bestguy** [comment]
Hi @koo5 , so this looks like we could support this with some changes:
https://svelte.dev/repl/a0503c2a3c6e47afbba89ffccbf76f3a?version=3.38.2

To do it consistently might take a bit of work, but in meantime you could temporarily workaround like so:

```
<button class="btn btn-primary" bind:this={my_button} on:click={more}>more...</button>
```
Using `btn` classes directly: https://getbootstrap.com/docs/5.0/components/buttons/

**Comment by christo** [comment]
For what it's worth I'm interested in this because I want to call focus and blur on <Input> components to enable inline editable components composed of a <span> when in uneditable mode and a sveltestrap <Input> when in edit mode. When the span is clicked the component hides the span and shows the Input but I want to then focus that input so I need a reference to that element even though it's not the element that received the event (so I can't use `event.target.focus()`)

**Comment by bestguy** [comment]
Hi @christo ,
We just added a solution for Input (#298) which was released in [v5.1.0](https://github.com/bestguy/sveltestrap/releases/tag/v5.1.0), but I think the same approach will work for Button and will look to add this week.

**Comment by bestguy** [comment]
Added in #303, should release soon

**Comment by bestguy** [comment]
Hi, this is released in [v5.1.1](https://github.com/bestguy/sveltestrap/releases/tag/v5.1.1)
Please reopen or comment if you see any issues.
